pkg/api/types.go

```diff
@@ -221,6 +224,25 @@ type GCEPersistentDiskVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty"`
 }
 
+// A ISCSI Disk can only be mounted as read/write once.
+type ISCSIDiskVolumeSource struct {
+	// iSCSI target portal
+	Portal string `json:"portal,omitempty" description:"iscsi target portal"`
```

Comment:
The word "portal" has specific meaning in kubernetes - can you either rename this or document that it does not necessarily mean the same thing as a kubernetes portal